src/components/manage/Blocks/Search/layout/LeftColumnFacets.jsx

```diff
@@ -96,18 +96,16 @@ const LeftColumnFacets = (props) => {
             </div>
           )}
 
-          <div>
```

Comment:
I think this div was needed for the flex layout to work well. Can we really remove it?

Reply:
@sneridagh Yes, we can safely remove that div.